task 1

```diff
@@ -3,4 +3,4 @@ def task
     myArray = ["stuff", "widget", "ruby", "goodies", "java", "emerald", "etc" ]
```

Comment:
this is 7th task, rename please file

Reply:
- You changed name of this file, but please add comment here that you made modifications
- Move all files to one directory called "Ruby Basic" or something like that